File: Java/04. Operation on String/72. Consonant or Vowel count letter.java
import java.util.Scanner;

class Main{
    public static void main(String[] args) {
        Scanner userInput = new Scanner(System.in);
        System.out.println("Enter a string to check it contains vowel:");
        String s = userInput.nextLine().toLowerCase();
        
        int count = 0;
        
        for (char chr : s.toCharArray()){
            if (chr=='a' || chr=='e' || chr=='i' || chr=='o' || chr=='u'){
                count++;
            }   
        }
        if (count>0){
            System.out.println(s + " has contains " + count + " Vowels and " + Math.abs(count-(s.length())) + " consonents.");
        }
        else {
            System.out.println(s + " is a complete Consonent Word");
        }
        userInput.close();
    }
}

File: Python/04. Operation on String/72. Consonant or Vowel count letter.py
s = input("Enter the String to Count the Vowel:").lower()
count = 0
for char in s:
    if (char=='a' or char=='e' or char=='i' or char=='o' or char=='u'):
        count += 1

if count>0:
    print(s, "has contains",count, "vowels and",abs(count-len(s)),"consonents.")
else:
    print("Given word has complete Consonent.")